sdks/builds/Makefile

```diff
@@ -44,7 +45,7 @@ all: package
 configure-mono: $(TOP)/configure
 
 $(TOP)/configure: $(TOP)/configure.ac $(TOP)/autogen.sh
-	cd $(TOP) && PATH=$(EXTRA_PATH):$$PATH NOCONFIGURE=1 ./autogen.sh
+	cd $(TOP) && PATH=$(EXTRA_PATH):$$PATH NOCONFIGURE=1 ./autogen.sh $(if $(wildcard $(TOP)/../mono-extensions),--enable-extension-module=xamarin --enable-extension-module)
```

Comment:
afaik we don't need `--enable-extension-module=xamarin`, just `--enable-extension-module`.

Reply:
We need both, see `mono-extension` repo for the rational.